src/Symfony/Component/HttpKernel/HttpKernel.php

```diff
@@ -113,6 +115,14 @@ public function terminateWithException(\Exception $exception)
      */
     private function handleRaw(Request $request, $type = self::MASTER_REQUEST)
     {
+        if (self::MASTER_REQUEST === $type) {
+            try {
+                // This will throw an exception if the headers are inconsistent.
```

Comment:
Maybe this comment is superfluous because the code is short and the exception message is very clear?